core/controllers/cron.py

```diff
@@ -223,3 +227,70 @@ def get(self):
             cron_services.JobModelsCleanupManager.enqueue(
                 cron_services.JobModelsCleanupManager.create_new())
             logging.warning('Deletion jobs for JobModels entities kicked off.')
+
+
+class CronMailReviewersContributorDashboardSuggestionsHandler(
+        base.BaseHandler):
+    """Handler for mailing reviewers suggestions on the Contributor
+    Dashboard that need review.
+    """
+
+    @acl_decorators.can_perform_cron_tasks
+    def get(self):
+        """Sends each reviewer an email with up to
+        suggestion_services.MAX_NUMBER_OF_SUGGESTIONS_TO_EMAIL_REVIEWER
+        suggestions that have been waiting the longest for review, based on
+        their reviewing permissions.
+        """
+        # Only execute this job if it's possible to send the emails.
+        if feconf.CAN_SEND_EMAILS and (
+                config_domain
+                .CONTRIBUTOR_DASHBOARD_REVIEWER_EMAILS_IS_ENABLED.value):
+            reviewer_ids = user_services.get_reviewer_user_ids_to_notify()
+            reviewers_suggestion_email_infos = (
+                suggestion_services
+                .get_suggestions_waiting_for_review_info_to_notify_reviewers(
+                    reviewer_ids))
+            email_manager.send_reviewers_contributor_dashboard_suggestions(
+                reviewer_ids, reviewers_suggestion_email_infos)
+
+
+class CronMailAdminContributorDashboardReviewIssuesHandler(
+        base.BaseHandler):
+    """Handler for mailing admins if there are issues that are causing a longer
+    reviewer turnaround time on the Contributor Dashboard.
+    """
+
+    @acl_decorators.can_perform_cron_tasks
+    def get(self):
+        """Sends each admin up to two emails: an email to alert the admins that
+        there are suggestion types that need more reviewers and/or an email
+        to alert the admins that specific suggestions have been waiting too long
+        to get reviewed.
+        """
+        if not feconf.CAN_SEND_EMAILS:
+            return
+        if config_domain.NOTIFY_ADMINS_REVIEWERS_NEEDED_IS_ENABLED.value:
+            admin_ids = user_services.get_user_ids_by_role(
+                feconf.ROLE_ID_ADMIN)
+            suggestion_types_need_reviewers = (
```

Comment:
suggestion_types_that_need_reviewers

Reply:
did suggestion_types_needing_reviewers for consistency from the other PR